smartseq2_single_sample/tasks/star.wdl

```diff
@@ -24,23 +23,68 @@ task Star {
       --outSAMunmapped Within \
       --limitBAMsortRAM 30000000000 \
       --outFileNamePrefix "${sample_tag}." \
+      --outSAMattributes All \
       --outSAMattrRGline ID:${id_tag} PL:illumina PU:${pu_tag} SM:${sample_tag} LB:${lib_tag}
     
-    mkdir logs
-    mv ${sample_tag}.Log.* ./logs/
-    tar -zcvf  ${sample_tag}.logs.tar.gz logs/
   }
   runtime {
-    docker:"humancellatlas/star_dev:2.5.3a"
+    docker:"humancellatlas/star:2.5.3a"
```

Comment:
The pure star image `humancellatlas/star:2.5.3a-1.0.0` has been pushed to Docker Hub, in case that you want to use it here.